build_tools/benchmarks/shark/run_shark.sh

```diff
@@ -72,4 +73,18 @@ echo "Benchmarks for IREE Complete"
 cat bench_results.csv
 mv bench_results.csv "${SHARK_OUTPUT_DIR}/${DRIVER}_iree_${IREE_VERSION}.csv"
 echo "######################################################"
+
+# Save version info.
+if [[ "${SAVE_VERSION_INFO}" = true ]]; then
```

Comment:
Nit: use double = (https://google.github.io/styleguide/shellguide.html#testing-strings)

```suggestion
if [[ "${SAVE_VERSION_INFO}" == true ]]; then
```

Reply:
Fixed